### PR TITLE
Create community content

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1688099](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1688099)

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles
- [x] Community content

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/community/community-support.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+# PowerShell community resources
+
+The PowerShell Community is a vibrant and active group of users. This article can help you get
+connected with other member of the community.
+
+## Getting support from the community
+
+The PowerShell community can file issues, bugs, or feature requests in our
+[GitHub](https://github.com/powershell/powershell/issues) repository. If you questions, you may find
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/community-support.md]: If you _have_ questions

File: reference/docs-conceptual/community/community-support.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+help from other members of the community in one of these public forums:
+
+- [User Groups](https://aka.ms/psusergroup)
+- [PowerShell Tech Community](https://techcommunity.microsoft.com/t5/PowerShell/ct-p/WindowsPowerShell)
+- [DSC Community](https://dsccommunity.org/)
+- [PowerShell.org](https://powershell.org/)
+- [StackOverFlow](https://stackoverflow.com/questions/tagged/powershell)
+- [r/PowerShell subreddit](https://www.reddit.com/r/PowerShell/)
+- [PowerShell Slack](http://poshcode.org/slack)
+- [PowerShell Discord](http://poshcode.org/discord)
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/community-support.md]: These should be https, but I see that pshcode.org has a cert problem. In the interim, might be best to use an aka.ms link to keep it as https, then switch back to poshcode.org once the cert issue is resolved.

File: reference/docs-conceptual/community/community-support.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+For more information about our support policy, see the
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/community-support.md]: I'd remove _more_, since the above isn't really part of the policy.

File: reference/docs-conceptual/community/contributing/editorial-checklist.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  - Change the date when there is a significant or factual update
+    - Reorganizing the article
+    - Fixing factual errors
+    - Adding new information
+  - Do not change the date if the update is insignificant
+    - Fixing typos and formatting
+- `title`: unique string of 43-59 chars including spaces
+  - Do not include site identifier (it is auto-generated)
+  - Use sentence case - capitalize only the first word and any proper nouns
+- `description`: 115-145 characters including spaces - this abstract displays in the search result
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/editorial-checklist.md]: Based on this, most of these new articles should have longer descriptions. 😄

File: reference/docs-conceptual/community/contributing/editorial-checklist.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  names, object or type names
+  - Bold is used for semantic markup, not emphasis
+  - Bold - use asterisks `**`
+- Italic - use underscore `_`
+  - Only used for emphasis, not for semantic markup
+- Line breaks at 100 columns (or at 80 for **about_Topics**)
+- No hard tabs - use spaces only
+- No trailing spaces on lines
+
+### Headers
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/editorial-checklist.md]: I thought of another to add here.  That headers should "gradually step down".  ie... no going straight down from H2 to H4.

File: reference/docs-conceptual/community/contributing/editorial-checklist.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+- Use [ATX Headers](https://github.github.com/gfm/#atx-headings) only
+- Use sentence case for all headers
+- Max depth of H3 or H4
+- Blank line before and after
+- PlatyPS enforces specific headers in its schema - do not add or remove headers
+
+### Code blocks
+
+- Blank line before and after
+- Use tagged code fences - **powershell**, **Output**, or other appropriate language ID
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/editorial-checklist.md]: Should we add a link here to other supported languages in the CG?

File: reference/docs-conceptual/community/contributing/editorial-checklist.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+- Untagged fence - syntax blocks or other shells
+- Put **Output** in separate code block except for simple examples where you don't intend the for
+  the reader to use the **Copy** button
+
+### Lists
+
+- Indented properly
+- Blank line before first item and after last item
+- Bullet - use hyphen "-" not asterisk "*" - too easy to confuse with emphasis
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/editorial-checklist.md]: Instead of quotes around those, should use the tick. The asterisk is unescaped. It renders fine, but formats in pure markdown oddly.

File: reference/docs-conceptual/community/contributing/editorial-checklist.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+- For numbered lists, all numbers are "1."
+
+## Terminology
+
+- PowerShell vs. Windows PowerShell vs. PowerShell Core
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/editorial-checklist.md]: This probably needs some additional explanation still.

File: reference/docs-conceptual/community/contributing/file-an-issue.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
+---
+title: How to file an issue
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/file-an-issue.md]: I'm wondering if this title should be more specific like 'How to file a docs issue'.

File: reference/docs-conceptual/community/contributing/file-an-issue.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+At the bottom of most pages on `docs.microsoft.com`, you'll see two feedback buttons. One is a link
+for product feedback and one is for documentation feedback. The documentation feedback requires a
+GitHub account. Clicking the button authenticates you in GitHub, then presents a simple form to
+enter your feedback. When you submit the form, the feedback control creates a new GitHub issue and
+links it to the current article.
+
+> [!NOTE]
+> This is not a support channel. This is a way to ask clarifying questions about documentation or
+> report errors and omissions. If you need technical support, see the
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/file-an-issue.md]: I'd rephrase to just `see [Community resources](../community-support.md).`

File: reference/docs-conceptual/community/contributing/get-started-writing.md [hunk 11]
```diff
@@ -0,0 +1,95 @@
+---
+title: Get started contributing to PowerShell documentation
+description: This article is an overview of how to get started as a contributor to the PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Get started contributing to PowerShell documentation
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/get-started-writing.md]: There should be some text under the H1.

File: reference/docs-conceptual/community/contributing/get-started-writing.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+The [PowerShell-Docs repository][psdocs] is divided into two groups of content. Git branches are
+used to manage how and when documentation gets published.
+
+### Reference content
+
+The reference content is the PowerShell cmdlet reference for the cmdlets that ship in PowerShell.
+The [reference][ref] is collected in versions folders (5.1, 6, 7.0, and 7.1). This content contains
+cmdlet reference only for the modules that ship with PowerShell. This content is also used to create
+the help information displayed by the `Get-Help` cmdlet.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/get-started-writing.md]: Should we have a link here to point to where to contribute to other ref content, such as Azure PowerShell?

File: reference/docs-conceptual/community/contributing/get-started-writing.md [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+- DSC documentation
+- SDK documentation
+
+The [conceptual documentation][conceptual] is not organized by version. All articles are displayed
+for every version of PowerShell. We are working to create version-specific articles. When that
+feature is available in our documentation, this guide will be update with the appropriate
+information.
+
+> [!NOTE]
+> Anytime a conceptual article is added, removed, or renamed, the TOC must be updated and included
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/get-started-writing.md]: Can we provide a link for what that TOC update looks like?

File: reference/docs-conceptual/community/contributing/get-started-writing.md [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+issues to make sure you're not duplicating efforts. Issues that are assigned to someone are
+considered to be "in progress". If you wish to collaborate on an issue, contact the person assigned
+to the issue.
+
+Similar to the PowerShell [RFC process][rfc], creating an issue before writing the content ensures
+that you don't spend a lot of time and effort on something that gets rejected by the PowerShell-Docs
+team. This also allows us to consult with you on the scope of the content and where is should fit
+within the PowerShell documentation.
+
+### Updating topics that existing articles
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/get-started-writing.md]: Feels like a word is missing from this title.

File: reference/docs-conceptual/community/contributing/managing-issues.md [hunk 15]
```diff
@@ -0,0 +1,93 @@
+---
+title: How we manage issues
+description: This article explains how the PowerShell-Docs team manages pull requests.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Managing Issues
+
+This article documents how we manage issues in the PowerShell-Docs repo. This article is designed to
+be a job aid for member of the PowerShell-Docs team. It is published here to provide process
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-issues.md]: _members_

File: reference/docs-conceptual/community/contributing/managing-issues.md [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+#### Label Types
+
+|Prefix  | Description                                                         |
+|------- | --------------------------------------------------------------------|
+|Area    | Used to indicate what part of PowerShell or the docs that the issue is discussing.<br>Useful for feature owners to find issues for their feature.|
+|Pri     | Used to indicate the priority of the issue. Value range 0-4.        |
+|Issue   | Used to classify the type of feedback for issue                     |
+|Review  | Used for issue that require further review by the team              |
+|Status  | Used to indicate the status of the work item                        |
+|Waiting | Used to indicate that we are waiting on                             |
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-issues.md]: Waiting on.... maybe add the words "external forces" or something to that effect.

File: reference/docs-conceptual/community/contributing/managing-issues.md [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+
+## Triage process
+
+The PowerShell docs team meets once per week to discuss any issues added since last meeting. An
+issue is considered to have been triaged when labels have been assigned or an owner has been
+assigned. PowerShell docs team members are encouraged to review the issues daily and triage new
+issues as they arrive. The weekly triage meeting can then be used to discuss the new issues in more
+detail, as needed.
+
+Misplaced product feedback
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-issues.md]: Was this supposed to be a header?

File: reference/docs-conceptual/community/contributing/managing-issues.md [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+  | DocSet    | Product Feedback URL                                         |
+  | --------- | ------------------------------------------------------------ |
+  | developer | https://github.com/PowerShell/PowerShell/issues/new/choose   |
+  | dsc       | https://windowsserver.uservoice.com/forums/301869-powershell |
+  | gallery   | https://github.com/powershell/powershellgallery/issues/new   |
+  | jea       | https://github.com/powershell/jea/issues/new                 |
+  | reference | https://github.com/PowerShell/PowerShell/issues/new/choose   |
+  | wmf       | https://windowsserver.uservoice.com/forums/301869-powershell |
+
+Support requests
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-issues.md]: Was this supposed to be a header?

File: reference/docs-conceptual/community/contributing/managing-issues.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+
+- If the support question is simple, answer it politely and close the issue.
+- If the question is more complicated, or the submitter replies with more questions, redirect them
+  to forums and support channels. Suggested text for redirecting to forums:
+
+    > This is not the right forum for these kinds of questions. Try posting your question in a
+    > community support forum. For a list of community forums see:
+    > https://docs.microsoft.com/powershell/scripting/community/community-support
+
+Code of conduct violations
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-issues.md]: Was this supposed to be a header?

File: reference/docs-conceptual/community/contributing/managing-pull-requests.md [hunk 20]
```diff
@@ -0,0 +1,112 @@
+---
+title: How we manage pull requests
+description: This article explains how the PowerShell-Docs team manages pull requests.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Managing pull requests
+
+This article documents how we manage pull requests in the PowerShell-Docs repo. This article is
+designed to be a job aid for member of the PowerShell-Docs team. It is published here to provide
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-pull-requests.md]: _members_

File: reference/docs-conceptual/community/contributing/managing-pull-requests.md [hunk 21]
```diff
@@ -0,0 +1,112 @@
...
+### Notes
+
+The following warnings can be ignored:
+
+- > Can't find service name for `<version>/<modulepath>/About/About.md`
+
+- > Metadata with following name(s) are not allowed to be set in Yaml header, or as file level
+  > metadata in docfx.json, or as global metadata in docfx.json: `locale`. They are generated by
+  > Docs platform, so the values set in these 3 places will be ignored. Please remove them from all
+  > 3 places to resolve the warning.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-pull-requests.md]: Would these be better in a codefence than a note bubble? The render looks odd (partial bubble off a line item).

File: reference/docs-conceptual/community/contributing/managing-pull-requests.md [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+
+- The `staging` branch should be merged to `live` at least once per week.
+- The `staging` branch should be merged to `live` after any large accumulation of changes.
+  - Changes to 50 or more files
+  - After merging a release branch
+- The `staging` branch should be merged to `live` after any significant change.
+  - Changes to repo or docset configurations (docfx.json, OPS configs, build scripts, etc.)
+  - Changes to the redirection file
+  - Changes to the TOC
+  - After merging a "project" branch (content reorg, bulk update, etc.)
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/managing-pull-requests.md]: This feels partially redundant to the previous line item. Can they be reworded and merged?

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md [hunk 23]
```diff
@@ -0,0 +1,492 @@
...
+description: This article provides the rules of style for writing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# PowerShell-Docs style guide
+
+This article provides style guidance specific to the PowerShell-Docs content. This builds on the
+information outlined in the [Overview](overview.md#get-started-writing-docs).
+
+## Product Terminology
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/powershell-style-guide.md]: This is the section I was looking for in the other doc.  A link here should suffice.

File: reference/docs-conceptual/community/contributing/editorial-checklist.md [hunk 24]
```diff
@@ -0,0 +1,112 @@
+---
+title: Editorial checklist
+description: This is a summarized list of rules for editing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Editor's checklist
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/editorial-checklist.md]: Do we also want to specify that files should always be lowercased and no special characters in the file names? It may be somewhere, but I don't recall seeing it and we should call it out.

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md [hunk 25]
```diff
@@ -0,0 +1,492 @@
...
+### Titles and headings
+
+Only use [ATX headings][atx] (# style, as opposed to `=` or `-` style headers).
+
+- Use sentence case - only proper nouns and the first letter of a title or header should be
+  capitalized
+- There must be a single space between the `#` and the first letter of the heading
+- Headings should be surrounded by a single blank line
+- Only one H1 per document
+- Header levels should increment by one. Do not skip levels
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/powershell-style-guide.md]: This is the skip comment I was looking for in the other doc.  This should probably be added there.

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md [hunk 26]
```diff
@@ -0,0 +1,492 @@
...
+image. Alternate text is required for screen readers for the visually impaired. It is also useful in
+case there is a site bug where the image cannot be rendered.
+
+Images should be stored in a `media/<article-name>` folder within the folder containing the article.
+Images should not be shared between articles. Create a folder that matches the filename of your
+article under the `media` folder. Copy the images for that article to that new folder. If an image
+is used by multiple articles, each image folder must have a copy of that image file. This practice
+prevents a change to an image in one article affecting another article.
+
+The following image file types are supported: *.png", *.gif", *.jpeg", *.jpg", *.svg
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/powershell-style-guide.md]: These should get code fenced or escaped due to the asterisk.  Also, they mostly end with quotes even though they didn't start with them.

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md [hunk 27]
```diff
@@ -0,0 +1,492 @@
...
+  ```markdown
+  By default, if you do not specify this parameter, the DMTF standard resource URI
+  `http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/` is used and the class name is appended to it.
+  ```
+
+- Links must have a friendly name, usually the title of the linked topic
+- All items in the "related links" section at the bottom should be hyperlinked.
+- Do not use backticks, bold, or other markup inside the brackets of a hyperlink.
+
+### Linking to other content
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/powershell-style-guide.md]: It was mentioned elsewhere, but do we want to add here that they should also be HTTPS?

File: reference/docs-conceptual/community/contributing/pull-requests.md [hunk 28]
```diff
@@ -0,0 +1,90 @@
...
+before submitting your pull request.
+
+## Target the correct branch
+
+All pull requests should target the `staging` branch. Changes should never be submitted to the
+`live` branch. Changes made in the `staging` branch get merged into `live`, overwriting any changes
+made to `live`.
+
+If you are submitting a change that only applies to an unreleased version of PowerShell, check for a
+release branch for that version. Your should be targeted at the release branch. Release branches
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/pull-requests.md]: Your _PR_

File: reference/docs-conceptual/community/contributing/pull-requests.md [hunk 29]
```diff
@@ -0,0 +1,90 @@
...
+must fix any errors or warnings (see exceptions) reported by the validation service.
+
+The following warnings can be ignored:
+
+- > Can't find service name for `<version>/<modulepath>/About/About.md`
+
+-  > Metadata with following name(s) are not allowed to be set in Yaml header, or as file level
+   > metadata in docfx.json, or as global metadata in docfx.json: `locale`. They are generated by
+   > Docs platform, so the values set in these 3 places will be ignored. Please remove them from all
+   > 3 places to resolve the warning.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/community/contributing/pull-requests.md]: Same comment, these should likely be codefenced instead of listitem to bubble.

File: reference/docs-conceptual/community/community-support.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/community-support.md]: fixed

File: reference/docs-conceptual/community/community-support.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/community-support.md]: fixed

File: reference/docs-conceptual/community/community-support.md (hunk 3 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/community-support.md]: fixed

File: reference/docs-conceptual/community/contributing/editorial-checklist.md (hunk 5 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/editorial-checklist.md]: added

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md (hunk 26 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/powershell-style-guide.md]: fixed

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md (hunk 27 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/powershell-style-guide.md]: It was a few like below but I added it here.

File: reference/docs-conceptual/community/contributing/pull-requests.md (hunk 28 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/pull-requests.md]: Fixed

File: reference/docs-conceptual/community/contributing/pull-requests.md (hunk 29 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/pull-requests.md]: Changed

File: reference/docs-conceptual/community/contributing/editorial-checklist.md (hunk 6 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/editorial-checklist.md]: Not public yet

File: reference/docs-conceptual/community/contributing/editorial-checklist.md (hunk 7 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/editorial-checklist.md]: Fixed

File: reference/docs-conceptual/community/contributing/editorial-checklist.md (hunk 8 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/editorial-checklist.md]: Linked to style guide

File: reference/docs-conceptual/community/contributing/file-an-issue.md (hunk 9 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/file-an-issue.md]: Fixed

File: reference/docs-conceptual/community/contributing/file-an-issue.md (hunk 10 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/file-an-issue.md]: Fixed

File: reference/docs-conceptual/community/contributing/get-started-writing.md (hunk 11 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/get-started-writing.md]: fixed

File: reference/docs-conceptual/community/contributing/get-started-writing.md (hunk 12 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/get-started-writing.md]: meh

File: reference/docs-conceptual/community/contributing/get-started-writing.md (hunk 13 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/get-started-writing.md]: link to public doc when available

File: reference/docs-conceptual/community/contributing/get-started-writing.md (hunk 14 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/get-started-writing.md]: fixed

File: reference/docs-conceptual/community/contributing/managing-issues.md (hunk 15 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-issues.md]: fixed

File: reference/docs-conceptual/community/contributing/managing-issues.md (hunk 16 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-issues.md]: something :)

File: reference/docs-conceptual/community/contributing/managing-issues.md (hunk 17 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-issues.md]: Fixed as H3

File: reference/docs-conceptual/community/contributing/managing-issues.md (hunk 18 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-issues.md]: H3

File: reference/docs-conceptual/community/contributing/managing-issues.md (hunk 19 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-issues.md]: H3

File: reference/docs-conceptual/community/contributing/managing-pull-requests.md (hunk 20 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-pull-requests.md]: fixed

File: reference/docs-conceptual/community/contributing/managing-pull-requests.md (hunk 21 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-pull-requests.md]: Fixed

File: reference/docs-conceptual/community/contributing/managing-pull-requests.md (hunk 22 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/managing-pull-requests.md]: Collapsed

File: reference/docs-conceptual/community/contributing/powershell-style-guide.md (hunk 23 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/powershell-style-guide.md]: doh

File: reference/docs-conceptual/community/contributing/editorial-checklist.md (hunk 24 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/editorial-checklist.md]: meh

File: reference/docs-conceptual/community/contributing/editorial-checklist.md (hunk 4 above)
Comment (sdwheeler) [reply reference/docs-conceptual/community/contributing/editorial-checklist.md]: stet
